src/redux/wallet/actions.js

```diff
@@ -59,6 +89,7 @@ export const transfer = (token: TokenModel, amount: string, recipient) => async
     dispatch(updateBalance(tokenDAO))
   } catch (e) {
     dispatch(showAlertModal({title: symbol + ' transfer error', message: e.message}))
+    throw e
```

Comment:
this is not what we have discussed
before this action-creator starts his work, we should already be sure (thanks to send form validation) that this transaction is possible, so we should update balance (subtract amount) before try-catch and we should update this balance back inside catch (add amount)
no more needed inside catch, so replace this two lines on balance rollback behaviour and manually update balance before try